priv/cabbage/apps/itest/lib/account.ex

```diff
@@ -79,9 +87,30 @@ defmodule Itest.Account do
     {:ok, address}
   end
 
+  defp hash(message), do: ExthCrypto.Hash.hash(message, ExthCrypto.Hash.kec())
+
   defp create_account_from_secret(secret, passphrase) do
-    Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], [])
+    if Application.get_env(:cabbage, :reorg) do
+      Reorg.create_account_from_secret(secret, passphrase)
+    else
+      with_retries(fn ->
+        Ethereumex.HttpClient.request("personal_importRawKey", [secret, passphrase], [])
+      end)
+    end
   end
 
-  defp hash(message), do: ExthCrypto.Hash.hash(message, ExthCrypto.Hash.kec())
+  def with_retries(func, total_time \\ 510, current_time \\ 0) do
```

Comment:
mixing private and public functions.
also, what is the reason we need to `retry`. Is it so that all nodes unlock an account? is there a better way to do this?

Reply:
I removed it. It was needed when accounts were unlocked in reorg

Reply:
I didn't find any other way. It is related to Nginx timeout. Some requests can no finish in 10 seconds. I set it 10 because during reorg Nginx continues to redirect to the paused node (til the first failure?), paused node accepts a request but do not return any response for N (timeout limit) seconds